docs/gitbook/architecture/public-private-control-model.md

```diff
@@ -0,0 +1,50 @@
+# Public/Private/Control Architectural Model
+
+The Interbit **Public/Private/Control (PPC)** architectural model is intended as a template for building Interbit applications that respect user's privacy.
+
+The PPC model is used within the [Interbit Accounts](https://accounts.interbit.io) so that each user will have their own private chain to ensure privacy of their personal data. It is also implemented in the *template* to enable developers to implement PPC applications that ensure user data privacy.
+
+The PPC model also enables *Chain Authorization* where users can authorize secure data sharing of their personal data in their private accounts chain with their private application chains.
+
+### Chain Roles within the PPC Model
+
+The PPC model defines specific roles for Public, Private and Control chains, but can be expanded to include chains for sharing data between users. The responsibilities of the specific chains in the PPC model are:
+
+**Public Chain**: Every user can load the public chain. The public chain contains the configuration required to create new private chains when a user launches the application.
+
+**Private Chain**: Each user has their own private chain for privacy, and only the owner can load and interact with that chain. The ACL of the private chain will be restricted, typically containing the public key(s) of the chain's owner and the chain ID of the control chain that created it.
+
+**Control Chain**: This is the administrative chain that manages the creation of private chains and is the source of truth of system data on the public chain (shared via a read join). Only administrators have access to this chain. Because this chain contain information on the private chains that it creates (join configuration or block history), it is a potential source of metadata leakage and should not be loaded on users browsers.
+
+**Shared Chains**: Many applications will require a chain or chains that *authorized* users can interact with by dispatching actions. Actions can be dispatched to this chain directly (if the user's public key is added to the chain's ACL) or indirectly (through a write join from the user's private chain). The shared chain is analogous to traditional shared public blockchains.
+
+
+## How the PPC model enables private user chains
+
+Private chains are created using *Chain Sponsorship*. If the user does not have a private chain in the browser's local storage, *Interbit Middleware* running on the browser can create a private chain and add the user's private key to the ACL of the new chain.
```

Comment:
There are concepts in this doc that have their own docs. Can we please link to them? E.g.
- chain sponsorship
- Interbit middleware
- chain joining